tools/__tasks__/compile/javascript/index.js

```diff
@@ -1,12 +1,12 @@
-module.exports = [{
-    title: 'Compile JS',
+module.exports = {
+    description: 'Compile JS',
     task: [
         require('./clean'),
-        require('../inline-svgs/compile'),
+        'compile/inline-svgs',
```

Comment:
yeah they are, but maybe that's not clear enough. i wonder if it offers too many ways of configuring?